extensions/amp-next-page/amp-next-page.md

```diff
@@ -163,82 +148,173 @@ Elements which are common across multiple pages can be programmatically
+| `image` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+
+To enable for an infinite loading experience without the need for a server-side configuration (remote-loading), `amp-next-page` automatically enables `deep-parsing` (see Attributes), which allows it to parse more suggestions by recursively looking at inline configurations inside `<amp-next-page>` tags on the loaded documents. To disable this behavior, set `deep-parsing` to `false`.
+
+#### Example Inline Configuration
 
-### Example configuration
+The following configuration will recommend two more documents for the user to read.
 
-The following configuration will recommend two more documents for the user to
-read, and hides the header and footer elements from each child document.
+```json
+[
+  {
+    "image": "https://example.com/image1.jpg",
+    "title": "This article shows first",
+    "url": "https://example.com/article1.amp.html"
+  },
+  {
+    "image": "https://example.com/image2.jpg",
+    "title": "This article shows second",
+    "url": "https://example.com/article2.amp.html"
+  }
+]
+```
+
+### Remote Configuration
+
+For remote configuration, the destination server is required to return a JSON object that has the following structure:
+
+| Key                | Value                                                                                                                                                                |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `pages` (required) | Array of pages, exactly the same format as the in the inline configration section above.                                                                             |
+| `next` (optional)  | Optional string url pointing to the next remote to query for more pages (should abide by the same rules as the initial URL, namely implement the CORS requirements.) |
+
+#### Example Remote Configuration
+
+The following configuration returned from the server will recommend two more documents for the user to read and tell `amp-next-page` to query `https://example.com/more-pages` when the user finishes reading the provided recommendations.
 
 ```json
 {
   "pages": [
     {
       "image": "https://example.com/image1.jpg",
       "title": "This article shows first",
-      "ampUrl": "https://example.com/article1.amp.html"
+      "url": "https://example.com/article1.amp.html"
     },
     {
       "image": "https://example.com/image2.jpg",
       "title": "This article shows second",
-      "ampUrl": "https://example.com/article2.amp.html"
+      "url": "https://example.com/article2.amp.html"
     }
   ],
-  "hideSelectors": [".header", ".footer"]
+  "next": "https://example.com/more-pages"
 }
 ```
 
+## Attributes
```

Comment:
I see this section doesn't include the `separator` attribute or `next-page-hide` attribute. Any particular reason for this?

Reply:
Hmm for `separator` it's an attribute on the child of `<amp-next-page>` and not on `<amp-next-page>` itself, while `next-page-hide` and `next-page-replace` are attributes on any AMP element that the user wants to hide so I don't think they fit here.